CHANGELOG.md

```diff
@@ -37,6 +38,7 @@
 * **Add title to Vectara upload - was not separated out from initial connector **
 * **Fix change OpenSearch port to fix potential conflict with Elasticsearch in ingest test **
 
+
```

Comment:
I don't believe we use double-blank-lines between releases. Check the others in this file and make this one consistent.